requirements.txt

```diff
@@ -1,4 +1,4 @@
-django>=2.0,<2.2
+django>=2.2,<2.2.1
```

Comment:
why are you making django 2.2 the minimum version? This shouldn't be done unless we decide so, but we didn't discuss this. Keep in mind for the future.

Reply:
I'll keep that in mind, also do I need to change it or is it okay for now?

Reply:
please revert this change